### DISCUSSION TITLE
OLED Connection

### DISCUSSION THREAD

**Discussion by souravlal** [marcel-licence/esp32_basic_synth#112] [Q&A]
Hi marcel license,
Please let me know to which pins of the ESP32 Audio Kit the i2c OLED display's(Scope display)  SCL and SDA pins are connected to? I look through the code but I couldn't find. Please help.

**Comment by marcel-licence** [comment]
Hi @souravlal ,

this will need some investigation. I thought that the I2C will be used in case the one connected to the ES8388 is also exposed to the GPIO's. Did you try that first?

**Reply by souravlal** [reply]
Thanks marcel license for the reply.
Do you mean io23 and io18, which are the i2c pins as per the ESP32 Audio kit pinout diagram on internet. Then which pins are used for i2c OLED scope display in your sampler project? Because both io23 and io18 are used for main st7735 display in that project.

**Reply by marcel-licence** [reply]
The ST7735 uses SPI. Please try the OLED at the I2C without the ST7735. I've added some information when using the Audio Kit which pins are used by the ES8388.
